addons/zeus/functions/fnc_moduleUnconscious.sqf

```diff
@@ -38,6 +38,10 @@ if (isNil QEFUNC(medical,setUnconscious)) then {
                 [LSTRING(OnlyAlive)] call FUNC(showMessage);
             } else {
                 private _unconscious = GETVAR(_unit,ACE_isUnconscious,false);
+                _unit setVariable [QGVAR(AIUnconsciousException), true, true];
+                    if (_unconscious) then {
+                    _unit setVariable [QGVAR(AIUnconsciousException), false, true];
+                    };
```

Comment:
```suggestion
                if (_unconscious) then {
                    _unit setVariable [QEGVAR(medical_statemachine,AIUnconsciousness), nil, true];
                } else {
                    _unit setVariable [QEGVAR(medical_statemachine,AIUnconsciousness), true, true];
                };
```

No point setting it to true if you're immediately going to change it.

Reply:
That is to prevent the situation where you would use the Zeus module on an AI dropping it to unconsciousness and then after a while, you would wake up the same AI again using the Zeus module, and then afterward, someone would shoot the AI into unconsciousness(With the AI unconsciousness turned off) which would not kill the AI as it should.